data_steward/tools/add_hpo.py

```diff
@@ -200,9 +248,38 @@ def bucket_access_configured(bucket_name):
         raise
 
 
+def update_site_masking_table():
+    """
+    Creates a unique `site_maskings` sandbox table and updates the `site_maskings` table with the
+        new site maskings
+    """
```

Comment:
maybe change the description since this is now just updating and the sandbox is being created in `generate_site_mappings_and_ext_tables`

Could be updated to:
```Updates the `pipeline_tables_sandbox.site_masking` sandbox table created in generate_site_mappings_and_ext_tables with any missing hpo_sites in lookup_tables.hpo_site_id_mappings```

Reply:
Yes, that makes sense! I will make that change and re-push!